app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +27,14 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
+      @repo = @deploy.repo
+      @pull_requests = Hubstats::PullRequest.belonging_to_deploy(@deploy.id)
```

Comment:
Use your association and write less code: `@pull_requests = @deploy.pull_requests`